ckanext/stadtzh_losdharvest/profiles.py

```diff
@@ -200,7 +199,9 @@ def _build_resources_dict(self, dataset_ref, dataset_dict):
             for key, predicate in (("created", SCHEMA.dateCreated),):
                 value = self._object_value(resource_ref, predicate)
                 if value:
-                    resource_dict[key] = self._clean_datetime(value)
+                    resource_dict[key] = self._format_datetime_as_isodate(
```

Comment:
I looked in the existing stadzh-harvester, and there, we don't add a `created` value to the resource_dict on harvesting it. As this field is overwritten by CKAN anyway, I think we can skip it in this harvester too, and get rid of the `_format_datetime_as_isodate` method. I think we should keep the renaming of `_format_datetime_as_string`, though, because it's clearer. :)